embc-app/ClientApp/src/app/store/registration/registration.reducer.ts

```diff
@@ -51,6 +51,7 @@ const initialState: State = {
     hasPersonalServicesReferral: null,
     hasChildCareReferral: null,
     hasPetCareReferral: null,
+    referralOther: null,
```

Comment:
This field is not needed. We already modelled it under the name **externalReferralsDetails** 
Please see Line 45 above

Reply:
Ok. I will change this.